jobs/registry-change.js

```diff
@@ -65,6 +72,22 @@ module.exports = async function (
     return
   }
 
+  let dependencies = [ dependency ]
+
+  // check if dependency update is part of a monorepo release
+  if (isPartOfMonorepo(dependency)) {
+    const version = distTags['latest']
+    if (force === true || !await hasAllMonorepoUdates(dependency, version)) {
+      log.info('exited: is not last in list of monorepo packages')
```

Comment:
Shouldn’t it be `force === false`? We _want_ this to continue if it is forced, right?